go.mod

```diff
@@ -51,7 +61,10 @@ replace (
 	k8s.io/kubernetes => k8s.io/kubernetes v1.21.1
 	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.21.1
 	k8s.io/metrics => k8s.io/metrics v0.21.1
+	k8s.io/mount-utils => k8s.io/mount-utils v0.21.2
 	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.21.1
+	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.9.2
```

Comment:
Do we really need `sigs.k8s.io/controller-runtime`?

Reply:
We are using it a lot under the PAO controller. In general, it makes it much easier to write a controller with it, without the need to specify separate clients for each API type, informers, and listers.

Reply:
It took me quite a bit of effort to get away from `operator-sdk` and `controller-runtime` in the past.  Perhaps things are better now, but it used to be quite hard (nearly impossible) to keep other k8s dependencies up-to-date when I was using those two dependencies in the past.  How easy/difficult is it now? I.e. use the current [1.22 k8s](https://github.com/openshift/cluster-node-tuning-operator/pull/261) deps?